nengo_viz/static/viz_slider.js

```diff
@@ -239,3 +345,9 @@ VIZ.Slider.prototype.layout_info = function () {
     info.max_value = this.scale.domain()[0];
     return info;
 };
+
+VIZ.Slider.prototype.update_value_text = function (slider_index, new_text) {
+    var self = this;
```

Comment:
For style consistency, this should be removed and just use ```this``` instead.

If we want to, we could go through all the code and use ```self``` everywhere (although I'd like to see the various reasons for and against that).  However, right now that's not the approach we've been doing, so we should keep the code style consistent.